packages/web/config.ts

```diff
@@ -30,4 +30,5 @@ export const CONFIG = {
     'https://d12-a-ceramic.3boxlabs.com',
   actionsURL: process.env.NEXT_ACTIONS_URL || 'http://localhost:4000',
   calendarId: 'nih59ktgafmm64ed4qk6ue8vv4@group.calendar.google.com',
+  googleDataApiKey: process.env.NEXT_PUBLIC_GOOGLE_API_KEY,
```

Comment:
```suggestion
  googleDataApiKey: process.env.NEXT_PUBLIC_YOUTUBE_API_KEY,
```

Since this is scoped just for the YouTube Data API

Reply:
ok